README.md

```diff
@@ -1,226 +1,49 @@
-    }
-  ],
-  "title": "Example Title",
-  "language": "http://lexvo.org/id/iso639-3/nno",
-  "startDate": "2018-11-01T00:00:00Z",
-  "endDate": "2021-05-01T00:00:00Z",
-  "funding": [
-    {
-      "type": "Funding",
-      "source": {
-        "type": "FundingSource",
-        "names": {
-          "nb": "Egen institusjon"
-        },
-        "code": "EI"
-      }
-    }
-  ],  
-  "coordinatingInstitution": {
-    "id": "https://api.cristin.no/v2/institutions/1234",
-    "type": "Organization",
-    "name": {
-      "en": "University"
-    }
-  },
-  "contributors": [
-    {
-      "type": "ProjectManager",
-      "identity": {
-        "id": "https://api.cristin.no/v2/persons/123456",
-        "type": "Person",
-        "firstName": "Ola",
-        "lastName": "Nordmann"
-      },
-      "affiliation": {
-        "id": "https://api.cristin.no/v2/institutions/1234",
-        "type": "Organization",
-        "name": {
-          "en": "University"
-        }
-      }
-    }
-  ],
-  "status": "CONCLUDED",
-  "academicSummary": {
-    "no": "<p>\r\n\tHelsedirektoratet har gjennom &rdquo;Underern&aelig;ringsprosjektet&rdquo; satt fokus p&aring; ern&aelig;ringssituasjonen p&aring; sykehus og institusjoner. Som et ledd i dette arbeidet skal gjeldende forskningsprosjekt unders&oslash;ke matservering/pasientbespisning ved sykehuset Ahus. M&aring;let for prosjektet er &aring; unders&oslash;ke sammenhengen mellom organiseringen av bespisningen og trivsel, matopptak og ern&aelig;ringssituasjon for pasientene. Fokus vil v&aelig;re p&aring; selve tjenesteleveransen og mindre p&aring; den ern&aelig;ringsmessige sammensetningen av mattilbudet. Bakgrunnen er at matserveringen p&aring; Ahus er et omr&aring;de som oppleves som problematisk av ansatte p&aring; flere av sykehusets mange avdelinger samt av ulike faggrupper og pasienter. Prosjektet gjennomf&oslash;res i samarbeid med Sykehuset Ahus og er tenkt knyttet til Helsedirektoratets satsing p&aring; forebyggende behandling av underern&aelig;ring.</p>\r\n"
-  },
-  "popularScientificSummary": {}  
-}
-```
-
-#### HTTP Status Codes lookup
-
-*   200 - Ok, returns one project.
-*   400 - Bad request, returned if the parameters are invalid.
-*   500 - Internal server error, returned if a problem is encountered retrieving project data
-*   502 - Bad Gateway, returned if upstream fetch fails or project not found
+Service for fetching data from the [Cristin API](https://api.cristin.no/v2/doc/index.html)
+
+### Building locally
+
+We need to add some tag exclusions to gradle when building locally because they are only used for 
+karate tests when deployed in AWS and will not work in a local environment.
+
+#### Windows
+
+```gradlew build -x karateTest -x createTestUsers -x deleteTestUsers```
+
+#### Linux
+
+```./gradlew build -x karateTest -x createTestUsers -x deleteTestUsers```
+
+### Running karate tests
+
+Karate tests are designed for being run in AWS via Github Actions as it needs a token that can
```

Comment:
Karate tests are designed to be run in AWS via Github Actions. This requires a token that can